src/export/CodeSystemExporter.ts

```diff
@@ -24,8 +24,8 @@ export class CodeSystemExporter {
     codeSystem.setId(fshDefinition);
     if (fshDefinition.title) codeSystem.title = fshDefinition.title;
     if (fshDefinition.description) codeSystem.description = fshDefinition.description;
-    // Version is set to value provided in config, will be overriden if reset by rules
-    codeSystem.version = this.tank.config.version;
+    delete codeSystem.version; // deleting to allow the IG Publisher default to take hold
+    codeSystem.status = this.pkg.config.status;
```

Comment:
The previous code used `this.tank.config` and this uses `this.pkg.config`.  Is there a meaningful difference?  I see that we're referencing `this.tank.config` in the line below, so if there isn't a difference, it might be best to use `this.tank.config` here too (for consistency).

Reply:
There's no difference, the config from the `tank` is actually used to initialize the `pkg`. Honestly there's no reason that I did this, I just referenced the config from the package out of habit :). I'll change that line here and in the other exporters for consistency!

Reply:
Addressed In commit [25e2afa](https://github.com/FHIR/sushi/pull/1143/commits/25e2afad1a873bf611c84ecca118ffc500f0d74e).